docker/cassandra-stress/Dockerfile-src

```diff
@@ -10,8 +10,8 @@ RUN git clone -b ${BRANCH} --single-branch ${GIT_FORK}
 RUN apt -y update && apt-get -y install ant
 
 # revert of cassandra driver (without shard-aware)
-COPY .gitconfig /root/
-RUN cd scylla-tools-java && git revert 1deceb8a605e563f01ef1821d48316cea766da31 && git revert 87a98f0a85c9db8a9c7602e66d3d2cca56903cdf
+# COPY .gitconfig /root/
```

Comment:
worth to mention what must happen to bring it back. If not, better remove.

Reply:
removed it